packages/web/src/app/search/page.tsx

```
@@ -1,39 +1,99 @@
+    (paper: Paper) =>
+      makeKeys(paper.doi, paper.title).some((k) => savedKeys.has(k)),
+    [makeKeys, savedKeys],
+  );
+
+  const markSaved = useCallback(
+    (paper: Paper) => {
+      const keys = makeKeys(paper.doi, paper.title);
+      if (keys.length === 0) return;
+      setSavedKeys((prev) => {
+        const next = new Set(prev);
+        keys.forEach((k) => next.add(k));
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)